src/Native/Web3.js

```diff
@@ -90,8 +90,8 @@ var _cmditch$elm_web3$Native_Web3 = function() {
                           + r.data
                           + "'})"
                     )
-                console.log(response)
-                return callback( succeed(response) );
+                console.log(response);
+                return callback( succeed({ ctor: "Bytes", _0: response }) );
```

Comment:
Since Bytes is now `type Bytes = Bytes String`, not returning a Bytes wrapped string caused things like Web3.Eth.estimateGas to return `undefined` as  `"aBytesStringNotABytesType"._0 === undefined`.